### ISSUE TITLE
transverse smile

### ISSUE THREAD

**Issue by CorneliusDreyling** [obophenotype/human-phenotype-ontology#10567]
**Preferred term label:**
transverse smile

**Synonyms**
horizontal smile

**Definition (free text, please give PubMed ID)**
Due to facial muscle weakness inability to raise the corners of the mouth. The act of smiling is accompanied by a horizontal movement of the mouth. (doi.org/10.1212%2FCON.0000000000000399)

**Parent term (use hpo.jax.org/app)**
Abnormality of mouth shape HP:0011338

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:269
**Your nano-attribution (ORCID)**
0009-0004-9942-3886

**Comment by pnrobinson** [comment]
@CorneliusDreyling 

Adding new term: 
- ID: HP:6000864
- label: Transverse smile
- parents: [Weakness of facial musculature (HP:0030319)](https://hpo.jax.org/app/browse/term/HP:0030319)
- pmid: PMID:25037087
- def: Inability to raise the corners of the mouth during the act of smiling due to facial muscle weakness. The act of smiling is accompanied by a horizontal movement of the mouth.
- comment: 
- orcid: orcid:0009-0004-9942-3886
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10567
- synonyms:
